src/modules/homeCounter.js

```diff
@@ -0,0 +1,10 @@
+const homeCounter = (length) => {
+  const homeIcon = document.getElementById('home-icon');
+
+  const counterSpan = document.createElement('span');
+  counterSpan.textContent = `(${length})`;
+
+  homeIcon.appendChild(counterSpan);
+};
+
+export default homeCounter;
```

Comment:
- [ ] Please make sure your counter functions make the counting based on what is actually displayed on the page instead of relying on the amount of data you get from the API. Also, make sure to cover all the edge cases you can think about.
- [ ] This applies to the comments counter you'll create as well. ✔

Reply:
Fixed.